barretenberg/cpp/src/barretenberg/plonk_honk_shared/execution_trace/execution_trace_usage_tracker.hpp

```diff
@@ -84,25 +84,23 @@ struct ExecutionTraceUsageTracker {
         for (auto [max_size, fixed_block] : zip_view(max_sizes.get(), fixed_sizes.get())) {
             size_t start_idx = fixed_block.trace_offset;
             size_t end_idx = start_idx + max_size;
-            active_ranges.push_back(Range{ start_idx, end_idx });
+            active_ranges.emplace_back(start_idx, end_idx);
         }
 
         // The active ranges must also include the rows where the actual databus and lookup table data are stored.
-        // (Note: lookup tables are constructed at the end of the trace; databus data is constructed at the start) so we
-        // need to determine the dyadic size for this. We call the size function on the current circuit which will have
-        // the same fixed block sizes but might also have an overflow block potentially influencing the dyadic circuit
-        // size.
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/1160)
-        const size_t dyadic_circuit_size = circuit.blocks.get_structured_dyadic_size();
+        // (Note: lookup tables are constructed from the beginning of the lookup block ; databus data is constructed at
+        // the start of the trace).
 
         // TODO(https://github.com/AztecProtocol/barretenberg/issues/1152): should be able to use simply Range{ 0,
-        // max_databus_size } but this breaks for certain choices of num_threads.
+        // max_databus_size } but this breaks for certain choices of num_threads. It should also be possible to have the
```

Comment:
Is your comment update still accurate? it looks like the ranges for tables are now constructed exactly as they should be

Reply:
Oh I see. I forgot we had separate ranges for the lookup block and the table data. I wonder if there is some problem in the algorithm that computes the reduced active ranges...

Reply:
double checking that would definitely be useful